assets/prototype/domain/shared/entities/tickets/predicates/isTrashed.ts

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
Looking at these 2 I am wondering if it is worth to have a common utility for both entities.
This way we will have to test just one predicate.
![image](https://user-images.githubusercontent.com/1477453/73353659-80b68c80-429c-11ea-8cc9-ed1be5bbaa74.png)

Reply:
Would be common for all I think

Reply:
would be nice if we could create a Type called something like :

```ts
type TrashableEntity = Datetime | Event | Price | Ticket;
```

but I don't think you can create types that are a set of interfaces.

Reply:
I like the second suggestion

Reply:
ya i should have just deleted the first idea... was kinda just writing everything out as things popped into my mind

Reply:
> ya i should have just deleted the first idea... was kinda just writing everything out as things popped into my mind



I like to follow the thought process :+1: